File: benchmark/results/java-invocation-inline/run-001/a-mcp/output/shipping/value_objects/DeliveryWindow.java
package com.metaengine.demo.shipping.value_objects;

import java.time.OffsetDateTime;

// DeliveryWindow value object.

public class DeliveryWindow {
    public String id;
    public OffsetDateTime createdAt;
    public OffsetDateTime updatedAt;
    public String name;
    public String description;

    public DeliveryWindow(String id, OffsetDateTime createdAt, OffsetDateTime updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }
}

File: benchmark/results/python-multilang/run-003/b-baseline/output/metaengine_demo/shipping/value_objects/delivery_window.py
"""DeliveryWindow value object for the shipping domain."""
from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class DeliveryWindow:
    """DeliveryWindow value object."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str
